dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/IdentifiableRepositoryImpl.java

```diff
@@ -891,6 +891,16 @@ public List<FileResource> setRelatedFileResources(
     return getRelatedFileResources(identifiableUuid);
   }
 
+  @Override
+  protected boolean supportsCaseSensitivityForProperty(String modelProperty) {
+    switch (modelProperty) {
+      case "label":
+        return true;
+      default:
+        return false;
```

Comment:
Müßte hier (und in allen anderen vergleichbaren Fällen) nicht eher ein `return super.supportsCaseSensitivityForProperty(modelProperty)` zurückgegeben werden, um die vererbten Properties richtig auszuwerten? So, wie es im `FileResourceMetadataRepositoryImpl` der Fall ist.

Reply:
Hier gibt's noch kein `super...`, weil das die erste nicht-abstrakte Klasse ist. Bei den Klassen, die von IdentifiableRepo abgeleitet sind, hab ich es mit `super` gemacht.